htdocs/master.inc.php

```diff
@@ -161,7 +161,7 @@ function is_countable($c)
 			exit;
 		}
 		dol_print_error($db, "host=".$conf->db->host.", port=".$conf->db->port.", user=".$conf->db->user.", databasename=".$conf->db->name.", ".$db->error);
-		exit;
+		exit(-1);
```

Comment:
With Dolibarr v20, negatives codes are now forbidden (not portable on some OS that convert negative coe into positive value).
Rule is return code 0 for no error and >= 1 when error.

So i switched your PR to exit(1)